posts/interaction-media-features.md

```diff
@@ -0,0 +1,11 @@
+feature: Interaction media features (hover, pointer, any-hover, any-pointer)
```

Comment:
I was copying the style of the "Viewport units (vh, vw, vmin, vmax)" entry.
Perhaps the `any-*`s should be dropped? Or maybe I should drop "Interaction media features"?